shared_kv_storage/Dockerfile

```diff
@@ -0,0 +1,98 @@
+# Copyright 2019 Intel Corporation

```

Comment:
Why your PR has changes to shared_kv_storage module? I think you need to re-base your PR.